utils/update_checkout/update-checkout-config.json

```diff
@@ -66,9 +74,13 @@
                 "swift-tools-support-core": "main",
                 "swiftpm": "main",
                 "swift-argument-parser": "0.4.3",
+                "swift-atomics": "0.0.3",
+                "swift-collections": "0.0.2",
                 "swift-crypto": "1.1.5",
                 "swift-driver": "main",
+                "swift-numerics": "0.1.0",
                 "swift-syntax": "main",
+                "swift-system": "0.0.1",
```

Comment:
One request, this tag is eight months old and doesn't build on Android, whereas latest trunk builds fine. Can you get @milseman to tag something more recent and use that instead?

Reply:
I'm looking at drafting a tag for this. Would a non-production release candidate work? We're looking at transitioning to a `1.0` and better/formalized release approach in the near future, but I want to unblock SPM ASAP.

Reply:
Since we're pre-`1.0`, I made a new release: https://github.com/apple/swift-system/releases/tag/v0.0.2

Reply:
updated to 0.0.2

Reply:
I removed the "v" prefix because it broke some of Github's tools (even though Github recommends it in their UI).

https://github.com/apple/swift-system/releases/tag/0.0.2